src/components/ReviewCard.jsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from "react-router-dom";
+
+export const ReviewCard = ({review_id, title, designer, owner, img_url, category, created_at, votes, review_body}) => {
+    return (
+        <div className="reviewCard">
+        <Link to={`/reviews/${review_id}`}>
+            <h3> {title} </h3>
+        </Link>
+            <img src={img_url} alt={`Image of review ${review_id}`} width="200" height="200" />
```

Comment:
No need to write 'image of' as we are already in an image. Try and be more specific. Also is the review_id a useful thing to tell the user that they are seeing?